superset-frontend/src/views/App.tsx

```diff
@@ -29,14 +27,13 @@ import { DynamicPluginProvider } from 'src/components/DynamicPlugins';
 import ErrorBoundary from 'src/components/ErrorBoundary';
 import Loading from 'src/components/Loading';
 import Menu from 'src/components/Menu/Menu';
-import FlashProvider from 'src/components/FlashProvider';
+import FlashProvider from 'src/components/FlashProvider/index';
```

Comment:
I this change necessary?

Reply:
When I removed the "/index" it started to point out an error in the typescript. I believe it is some configuration in the webpack.

Reply:
That's strange we have quite a few modules using this pattern already. You probably needed to restart your webpack server as it's probably pointing to an outdated module resolution cache for this file (happens when you don't change the path but do change the file structure).

Reply:
I cloned the repository again and it worked